back/doc/dependency_decisions.yml

```diff
@@ -175,6 +175,13 @@
 - - :approve
   - bundler-audit
   - :who: 
-    :why: Despite bundler-audit being a GPL utility, we're not linking against it, but merely executing it in its distributed form as a rake task.
+    :why: Despite bundler-audit being a GPL utility, we're not linking against it,
+      but merely executing it in its distributed form as a rake task.
     :versions: []
     :when: 2022-06-17 14:33:30.806575129 Z
+- - :approve
+  - customizable_navbar
+  - :who: 
+    :why: 
+    :versions: []
+    :when: 2022-10-11 20:41:20.553607345 Z
```

Comment:
I've never seen this file before. What does it do?

Reply:
It says we allow the new dependency, `customizable_navbar`. We have the same for other free engines in this file.